Unreal/SystemLocalized/det/UnrealShare.det

```diff
@@ -1358,7 +1358,7 @@ M_HintMessage="Neue Hinweismeldung (zum Lesen F3 drücken)."
 ; EN: DeathMessage="%o got gatted by %k's %w."
```

Comment:
Another uneditable batch, some remarks:

Line 1266: "KONTROLLEN" → "STEUERUNG"

Line 1271: "Schiessen" → "Schießen"

Line 1315: "Admin Menu Anzeigen" → "Admin Menü Anzeigen"

Reply:
@SlyMk3 That substitution for 1315 doesn't look any better. Applied my own take.